templates/medias/details.rs.html

```diff
@@ -0,0 +1,34 @@
+@use templates::base;
+@use template_utils::*;
+@use plume_models::medias::Media;
+
+@(ctx: BaseContext, media: Media)
+
+@:base(ctx, "Media details", {}, {}, {
+    <h1>@i18n!(ctx.1, "Media details")</h1>
+    <section>
+        <a href="/medias">@i18n!(ctx.1, "Go back to the gallery")</a>
+    </section>
+
+    <section>
+        <figure class="media">
+            @Html(media.html(ctx.0))
+            <figcaption>@media.alt_text</figcaption>
+        </figure>
+        <div>
+            <p>
+                @i18n!(ctx.1, "Markdown code")
+                <small>@i18n!(ctx.1, "Copy it in your articles to insert this media.")</small>
+            </p>
+            <code>@media.markdown(ctx.0)</code>
+        </div>
+        <div>
+            <form class="inline" method="post" action="/medias/@media.id/avatar">
```

Comment:
this button should not be here if MediaCategory != Image
(was true for Tera too though)